src/Umbraco.Forms.Integrations.Crm.ActiveCampaign/Client/src/property-editor/manifests.ts

```diff
@@ -1,9 +1,8 @@
-import { ManifestTypes } from "@umbraco-cms/backoffice/extension-registry";
 import { manifest as accountManifest } from "./account/manifests";
 import { manifest as contactMappingManifest } from "./contact-mapping/manifests";
 import { manifest as customMappingManifest } from "./custom-mapping/manifests";
 
-export const manifests : Array<ManifestTypes> = [
+export const manifests = [
```

Comment:
Think you should still have the type of the array here, but it needs to be:

```
export const manifests: Array<UmbExtensionManifest> = [
```

Have a read of [this](https://github.com/umbraco/Announcements/issues/22) and particularly under "Typescript configuration" for the details here of how to reference this type.